### PR TITLE
Added event to log errors on the bot

### PR DESCRIPTION
**Pull request by DylanD16**
This should help me, in particular, to submit a PR to fix some of those issues with the mute commands.

### CODE REVIEW

**Comment by GavynStanley** [comment]
I should've added this anyways. What channel does this log to?

**Comment by DylanD16** [comment]
It logs to #error-logs which is the same channel as the site logs

File: cogs/events.py [hunk 1]
```diff
@@ -43,6 +43,19 @@ async def on_command_error(self, ctx, error):
...
                                description = f"`{str(error.param).partition(':')[0]}` is a required argument!")
             await ctx.send(embed = em)
+                               
+    @commands.Cog.listener('on_command_error')
+    async def log_error(self, ctx, error):
+        if isinstance(error, commands.CommandNotFound):
+            return
+        channel = self.bot.get_channel(728994664631500940)
+        em = discord.Embed(
+            title='Error:',
```
Comment (GavynStanley) [inline cogs/events.py]: Have it say Bot Error so it doesn't conflict with site errors format

**Review by GavynStanley — CHANGES_REQUESTED** [review]

File: cogs/events.py (hunk 1 above)
Comment (DylanD16) [reply cogs/events.py]: Alright

File: cogs/events.py [hunk 2]
```diff
@@ -43,6 +43,19 @@ async def on_command_error(self, ctx, error):
             em = discord.Embed(color = discord.Color.red(),
                                description = f"`{str(error.param).partition(':')[0]}` is a required argument!")
             await ctx.send(embed = em)
+                               
+    @commands.Cog.listener('on_command_error')
```
Comment (GavynStanley) [inline cogs/events.py:47]: Why do this when you could make the function name `on_command_error`

Comment (DylanD16) [reply cogs/events.py:47]: It would clash with the error handler directly above and I think its good to split the error logs from the error messages

**Pull request merged**